src/templates/en/2020/methodology.html

```diff
@@ -104,7 +108,7 @@ <h2 id="dataset"><a href="#dataset" class="anchor-link">About the dataset</a></h
         </div>
```

Comment:
Should we update this example SQL by picking one from the 2020 Almanac?

Reply:
+1 when more queries are merged it'd be nice to update this to 2020.

Reply:
Resolving this for now but we can revisit before launch.